src/components/Breadcrumbs.tsx

```diff
@@ -91,21 +50,51 @@ const Breadcrumbs: React.FC<IProps> = ({
   })
 
   return (
-    <ListContainer aria-label="Breadcrumb" dir="auto" {...restProps}>
-      <List>
+    <Box
+      aria-label="Breadcrumb"
+      dir="auto"
+      {...restProps}
+      as="nav"
+      marginBottom={8}
+      listStyleType="none"
+      position="relative"
+      zIndex={1}
+    >
+      <UnorderedList
+        display="flex"
+        flexWrap="wrap"
+        listStyleType="none"
+        margin={0}
+      >
         {crumbs.map((crumb, idx) => (
-          <ListItem key={idx}>
-            <CrumbLink
+          <ListItem
+            key={idx}
+            margin={0}
+            marginRight={2}
+            fontSize="sm"
+            lineHeight="140%"
+            letterSpacing="wider"
+          >
+            <ButtonLink
```

Comment:
This should just be the `Link` component, as the crumbs only need to be links and not also button. (There is also the solid background from the default button styling that should not be there)